advocacy_docs/partner_docs/ThalesGuide/04-ImplementingCTE.mdx

```diff
@@ -35,25 +33,30 @@ sudo yum install -y lsof
   <img  src="Images/CipherTrustManager.png">
 </p>
 
-### Configuring CipherTrust Manager
+### 3.2 Configuring CipherTrust Manager
 Logon to the CipherTrust Manager (CM) Web GUI and perform the following steps:
 
-1. **Create  Registration Token**
+1. Create  Registration Token
 
-    a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token will be used for the CTE agent enrollment to CM. 
+  a. Navigate to **Key and Access Management** and select **Registration Tokens**. This token will be used for the CTE agent enrollment to CM. 
```

Comment:
@josh-heyer, this sub-step is not wrapping correctly. Note sure what the issue is.